third_party/ampli/ampli.go

```diff
@@ -0,0 +1,527 @@
+// ampli.go
+//
+// Ampli - A strong typed wrapper for your Analytics
+//
+// This file is generated by Amplitude.
+// To update run 'ampli pull ddev'
```

Comment:
Link to docs would be great here.

Reply:
This file is generated, we should not manually add something here. But I've added some hint to the docs already.